src/lib/datepicker/calendar.html

```diff
@@ -36,6 +36,8 @@
       [activeDate]="_activeDate"
       [selected]="selected"
       [dateFilter]="_dateFilterForViews"
```

Comment:
I think you can remove `_dateFilterForViews` now and just use the date filter that was passed to the calendar